gateway/modules/schema/schema.go

```diff
@@ -153,15 +153,28 @@ func getCollectionSchema(doc *ast.Document, dbName, collectionName string) (mode
 					switch directive.Name.Value {
 					case model.DirectivePrimary:
 						fieldTypeStuct.IsPrimary = true
+						fieldTypeStuct.PrimaryKeyInfo = &model.TableProperties{}
 						for _, argument := range directive.Arguments {
 							if argument.Name.Value == "autoIncrement" {
 								val, _ := utils.ParseGraphqlValue(argument.Value, nil)
 								switch t := val.(type) {
 								case bool:
-									fieldTypeStuct.IsAutoIncrement = t
+									fieldTypeStuct.PrimaryKeyInfo.IsAutoIncrement = t
 								default:
 									return nil, helpers.Logger.LogError(helpers.GetRequestID(context.TODO()), fmt.Sprintf("Unexpected argument type provided for field (%s) directinve @(%s) argument (%s) got (%v) expected string", fieldTypeStuct.FieldName, directive.Name.Value, argument.Name.Value, reflect.TypeOf(val)), nil, map[string]interface{}{"arg": argument.Name.Value})
 								}
+
+							}
+							if argument.Name.Value == "order" {
+								val, _ := utils.ParseGraphqlValue(argument.Value, nil)
+								t, ok := val.(int)
```

Comment:
I thought parseGraphqlValue returns int64

Reply:
It is int only
![image](https://user-images.githubusercontent.com/24411676/103193353-369db700-4902-11eb-8ad9-afbcb201a78b.png)